README.md

```diff
@@ -79,25 +84,32 @@ It's likely that the Service Bus topic and subscription names will need to be am
 > Note: if you wish to run this service end to end with [Payment Responses](https://github.com/DEFRA/ffc-pay-responses), then update the `BATCH_STORAGE` environment variable to use port `10002` instead of `10007`.
 
```

Comment:
Would remove following paras from Configuration

For blob, examples assumes option 1 is taken above and therefore shows connection strings for local Azurite container.

It's likely that the Service Bus topic and subscription names will need to be amended to match those owned by the developer.